README.md

```diff
@@ -29,7 +29,8 @@ Key features
 * Uses the new Airbrake JSON
   (v3)<sup>[[link](https://airbrake.io/docs/#create-notice-v3)]</sup>
 * SSL support (all communication with Airbrake is encrypted by default)
-* Support for .NET 3.5 and above (including the latest .NET Core platforms)
+* Support for .NET 4.5.2 and above (including the latest .NET Core platforms)
+  <sup>[[net35](#net-35-support)]</sup>
```

Comment:
That link is to clarify what is going on with support for .NET 3.5 and 4.5. It resolves to the section ".NET 3.5 Support".